st2actions/st2actions/utils/param_utils.py

```diff
@@ -30,8 +30,10 @@
 
 
 def _split_params(runner_parameters, action_parameters, mixed_params):
-    pf = lambda params, skips: {k: v for k, v in six.iteritems(mixed_params)
-                                if k in params and k not in skips}
+    def pf(params, skips):
```

Comment:
no love for lambdas? Did this get us into pep8 trouble or is it personal preference?